app/controllers/partners/invitations_controller.rb

```diff
@@ -0,0 +1,50 @@
+# This exists so that we can override some of the devise resource
+class Partners::InvitationsController < Devise::InvitationsController
+  layout "devise"
+
+  skip_before_action :authorize_user
+  skip_before_action :authenticate_user!
+  # This one causes a redirect require_no_authentication
+  skip_before_action :require_no_authentication
+
+  def update
+    raw_invitation_token = update_resource_params[:invitation_token]
+    self.resource = accept_resource
```

Comment:
what is `accept_resource`? is that a devise thing?

Reply:
nm I realized what was going here in a later comment

Reply:
@albertchae I'll add comments to explain these parts and also indicate where it came from :)